[WebExtensions] Deprecate runtime.PlatformInfo.nacl_arch (#43799)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/runtime/platforminfo/index.md

  - : {{WebExtAPIRef('runtime.PlatformOs')}}. The platform's operating system.
- `arch`
  - : {{WebExtAPIRef('runtime.PlatformArch')}}. The platform's processor architecture.
- `nacl_arch` {{deprecated_inline}}

>   - : {{WebExtAPIRef('runtime.PlatformNaclArch')}}. The Google Native Client architecture. This attribute was never supported by non-Chromium browsers and is in the process of removal from Chromium. Consider migrating to `arch`, which contains equivalent information and is more descriptive on some platforms (ARM and RISC-V).

Reviewer: ```suggestion
  - : {{WebExtAPIRef('runtime.PlatformNaclArch')}}. The Google Native Client architecture. Only Chromium-based browsers support this attribute, and Chromium is removing it. Consider migrating to `arch`, which contains equivalent information and is more descriptive on some platforms (ARM and RISC-V).
```
